test/E2ETest/TyeRunTests.cs

```diff
@@ -291,7 +291,7 @@ public async Task FrontendBackendWatchRunTest()
                         return;
                     }
 
-                    await Task.Delay(500);
+                    await Task.Delay(5000);
```

Comment:
Why?

Reply:
test was failing on CI, pretty much need to wait for dotnet build which apparently takes longer than 5 seconds

Reply:
Yep!! Or we could wait for the replica to be up again after going down.

Reply:
Yeah, potentially if this is flaky.